lib/themes/dosomething/paraneue_dosomething/includes/helpers.inc

```diff
@@ -344,6 +344,40 @@ function paraneue_dosomething_get_themed_placeholder_reportbacks($count = 6) {
 }
 
 /**
+ * Get regional url data for specified node if it has translated content available.
+ *
+ * @param object $node  Specified Node object for which to add regional urls.
+ * @return array|null
+ */
+function paraneue_dosomething_get_regional_url_data($node) {
+  global $language_url;
+  global $base_url;
+
+  $available_translations = dosomething_helpers_isset($node->translations->data);
+
+  if (!$available_translations) {
+    return NULL;
+  }
+
+  $supported_languages = variable_get('dosomething_global_language_map');
+  $regional_url_data = [];
+
+  foreach ($available_translations as $language_key => $translation) {
+    // Skip creating regional url for global english.
+    if ($language_key === 'en-global') {
+      continue;
+    }
+
+    if (array_key_exists($language_key, $supported_languages) && $language_key !== $language_url->language) {
+      $country = strtolower($supported_languages[$language_key]['country']);
```

Comment:
country won't be mapped if we use language_list, so there's a convert to language function in `dosomething_global`